### PR TITLE
Ebp info

### PR DESCRIPTION
**Pull request by mpoelchau**

### CODE REVIEW

File: _pages/main.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ type: single
 permalink: /
 ---
 
+
+## Join the 'Insects' breakout session at this year's G10K-VGP EBP 2019 Meeting! This year's meeting is a joint effort between the Vertebrate Genomes Project (VGP) and the Earth BioGenome Project (EBP). The discussion-based insects breakout session is scheduled for 9am -12 pm Eastern, Wednesday, August 28th. If you are not coming to the meeting, but are interested in shaping the discussion about insects or arthropods in the Earth BioGenome Project, we encourage you to attend the session remotely.
+* [Read the Insects breakout session agenda](ebp_insects_agenda)
```
Comment (childers) [inline _pages/main.md]: This gave me a 404 error when I tested it on my local.  
ebp_insects_agenda.md is the page.  I

File: _pages/ebp_insects_agenda.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+**Earth BioGenome meeting 2019 - Insects Session**  **agenda**
```
Comment (childers) [inline _pages/ebp_insects_agenda.md:6]: We need to add a header to let Jekyll know what this page is about.

```
---
title: "The Insects breakout session for the Earth BioGenome 2019 meeting"
type: single
permalink: /ebp_insects_agenda
---
```

**Review by childers — CHANGES_REQUESTED** [review]
I think all we need to change is adding a header to the new page. I'll test on my local and update you.

File: _pages/ebp_insects_agenda.md (hunk 2 above)
Comment (childers) [reply _pages/ebp_insects_agenda.md:6]: Adding this fixed the linking issue form the main page.

Comment (mpoelchau) [reply _pages/ebp_insects_agenda.md:6]: Awesome, thanks!

**Review by childers — APPROVED** [review]

**Review by beedatadriven — APPROVED** [review]

**Pull request merged**